app/models/consultation/interaction.rb

```diff
@@ -0,0 +1,2 @@
+class Consultation::Interaction < Consultation::Base
```

Comment:
This file is created as part of Step 4 in [Creating a new data source](https://github.com/metridoc/metridoc-rails/wiki/Creating-a-new-data-type).